package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "airbender",
+  "version": "0.0.1",
+  "description": "<p align=\"center\"> <a href=\"https://dscvit.com\"> \t<img src=\"https://user-images.githubusercontent.com/30529572/72455010-fb38d400-37e7-11ea-9c1e-8cdeb5f5906e.png\" alt=\"DSC VIT\"/> </a> \t<h2 align=\"center\"> Airbender </h2> \t<h4 align=\"center\"> Draw and highlight user avatars in Discord Voice Channel <h4> </p>",
+  "main": "index.js",
+  "dependencies": {
+    "discord.js": "^12.3.1",
+    "opusscript": "^0.0.7",
+    "typescript": "^4.0.2"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Run script should be here as `start`.

Reply:
The run script will spawn the GUI - since that isn't made yet, I haven't mentioned any scripts in this section